src/Util/ClassSourceManipulator.php

```diff
@@ -293,6 +363,12 @@ public function addAnnotationToClass(string $annotationClass, array $options)
         $this->updateSourceCodeFromNewStmts();
     }
 
+    public function clearClassNodeStmts()
+    {
+        $this->getClassNode()->stmts = [];
+        $this->updateSourceCodeFromNewStmts();
+    }
```

Comment:
Hopefully this won't be needed at the end :)

Reply:
Yes :)